subsys/usb/usb_c/usbc_pe_common.c

```diff
@@ -19,6 +19,19 @@ LOG_MODULE_DECLARE(usbc_stack, CONFIG_USBC_STACK_LOG_LEVEL);
 
 static const struct smf_state pe_states[PE_STATE_COUNT];
 
+/**
+ * @brief Set the ready state for sink or source.
+ */
+static void pe_set_ready_state(const struct device *dev)
+{
+	struct usbc_port_data *data = dev->data;
+
+	if (data->pe->power_role == TC_ROLE_SOURCE)
+		pe_set_state(dev, PE_SRC_READY);
+	else
+		pe_set_state(dev, PE_SNK_READY);
```

Comment:
Add curly braces as suggested by clang-fomat here.

Reply:
fixed, was not aware that was an exception from the linux kernel style.
but i prefer to always use curly braces so it's a good exception.